katas/content/teleportation/reconstruct_and_measure_message/solution.md

```diff
@@ -0,0 +1,8 @@
+The exercise can be done easily using previous solutions with slight modification.
+- Reconstruct `qBob` qubit using the ReconstructMessage operation from exercise. This would bring the qBob state into one of the eigenstates depending to the basis mentioned.
```

Comment:
```suggestion
First, reconstruct the state of qubit `qBob` using the `ReconstructMessage` operation from an earlier exercise. This would bring the `qBob` state into one of the basis states of the given basis.
```